e2e/hive/simulators/graphql/testcases/12_eth_getBalance_toobig_bn.json

```diff
@@ -0,0 +1,15 @@
+{
+  "request": "{block (number: 33) {account(address: \"0x6295ee1b4f6dd65047762f924ecd367c17eabf8f\") { balance } }}",
+  "responses": [
+    {
+      "data": {
+        "block": {
+          "account": {
+            "balance": "0x0"
```

Comment:
are these from your tests or fromt he graphql tests for mainnet (like in the hive repo).

Reply:
hive repo